Lesson2/src/main/java/pl/slvd/university/Deanery.java

```diff
@@ -0,0 +1,43 @@
+package pl.slvd.university;
+
+import java.util.*;
+import java.util.stream.IntStream;
+
+class Deanery implements Comparator<Applicant> {
+
+    @Override
+    public int compare(Applicant o1, Applicant o2) {
+        return o2.getSum() - (o1.getSum());
+    }
+
+    public static void sortByGrades(List<Applicant> sortedList, String firstLastName, int getNumOfBudgetPlaces, int getNumOfPaidPlaces) {
+        System.out.println("Ready to see the lists of enrolled in the University?");
```

Comment:
maybe print some info for users with possible input?

Reply:
Yes, I missed it. I will display information on the number of places in the Speciality.